.env.stable

```diff
@@ -2,8 +2,8 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-PFCP_AGENT_IMAGE=omecproject/upf-epc-pfcpiface:master-f3a2c33
-ONOS_IMAGE=opennetworking/sdfabric-onos:master-2022-03-05
+PFCP_AGENT_IMAGE=omecproject/upf-epc-pfcpiface:master-762b242
+ONOS_IMAGE=dmoro92/sdfabric-onos:slice_meter
```

Comment:
is this ok ?

Reply:
Yes, unfortunately, we can't build a `sdfabric-onos` image right now with UP4 master (due to changes in ONOS APIs). For this reason, I had to build a custom `sdfabric-onos` image and publish that under my docker. I plan to merge it as is, trigger a sdfabric-onos build and then change this right after the image is published.